array-stack-queue/1.c

```diff
@@ -0,0 +1,16 @@
+#include<stdio.h>
```

Comment:
Please Rename this file and give it a revealing name. Same thing for your other files